src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+
+/** Association class to support the following functionality:
+ * When the AddressBook program exits, it should print out a list of all the tags added/deleted during that session.
```

Comment:
A more precise description is `Represents an adding or removing a Tag to/from a Person`.